x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/customization_modal.tsx

```diff
@@ -0,0 +1,159 @@
+
+export const CustomizationModal: React.FC<Props> = ({
+  filterFields,
+  onClose,
+  onSave,
+  sortFields,
+}) => {
+  const { engine } = useValues(EngineLogic);
+
+  const [selectedFilterFields, setSelectedFilterFields] = useState(
+    filterFields.map(fieldNameToComboBoxOption)
+  );
+  const [selectedSortFields, setSelectedSortFields] = useState(
+    sortFields.map(fieldNameToComboBoxOption)
+  );
+
+  const engineSchema = engine.schema || {};
+  const selectableFilterFields = Object.keys(engineSchema).map(fieldNameToComboBoxOption);
+  const selectableSortFields = Object.keys(engineSchema).map(fieldNameToComboBoxOption);
+
+  return (
+    <EuiOverlayMask>
+      <EuiModal onClose={onClose}>
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.title',
+              {
+                defaultMessage: 'Customize document search',
+              }
+            )}
+          </EuiModalHeaderTitle>
+        </EuiModalHeader>
+        <EuiModalBody>
+          <EuiForm>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.filterFieldsLabel',
+                {
+                  defaultMessage: 'Filter fields',
+                }
+              )}
+              fullWidth={true}
+              helpText={
+                <EuiText>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.filterFields',
+                    {
+                      defaultMessage:
+                        'Faceted values rendered as filters and available as query refinement',
+                    }
+                  )}
+                </EuiText>
+              }
+            >
+              <EuiComboBox
+                data-test-subj="filterFieldsDropdown"
+                fullWidth={true}
+                options={selectableFilterFields}
+                selectedOptions={selectedFilterFields}
+                onChange={setSelectedFilterFields}
+              />
+            </EuiFormRow>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.sortFieldsLabel',
+                {
+                  defaultMessage: 'Sort fields',
+                }
+              )}
+              fullWidth={true}
+              helpText={
+                <EuiText>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.sortFields',
+                    {
+                      defaultMessage:
+                        'Used to display result sorting options, ascending and descending',
+                    }
+                  )}
+                </EuiText>
+              }
```

Comment:
This text already has a `.euiFormHelpText` class as a wrapper, it doesn't need an `<EuiText>` as well just IMO (makes the font uncomfortably large.

```suggestion
              helpText={i18n.translate(
                'xpack.enterpriseSearch.appSearch.documents.search.customizationModal.filterFields',
                {
                  defaultMessage:
                    'Faceted values rendered as filters and available as query refinement',
                }
              )}
```

Reply:
eb4292447dbb23bef8680270372fe54eac20a93c

Reply:
eb4292447dbb23bef8680270372fe54eac20a93c